src/test/javascript/spec/component/plagiarism/plagiarism-cases-instructor-view.component.spec.ts

```diff
@@ -25,6 +33,8 @@ describe('Plagiarism Cases Instructor View Component', () => {
     let component: PlagiarismCasesInstructorViewComponent;
     let fixture: ComponentFixture<PlagiarismCasesInstructorViewComponent>;
     let plagiarismCasesService: PlagiarismCasesService;
+    let router: MockRouter;
+    const originalConsoleWarn = console.warn;
```

Comment:
Please add a comment on why you need `const originalConsoleWarn = console.warn;` here.

Reply:
this was needed to disable warnings about ambiguous element selection due to duplicate elements in test, fixed code so that it is not needed.